docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,390 +1,565 @@
-stopPosition.Add(new PropertyValue(0.8f));
-stopPosition.Add(new PropertyValue(1.0f));
-gradientVisualMap1.StopOffset = stopPosition;
 
-PropertyArray stopColor = new PropertyArray();
-stopColor.Add(new PropertyValue(new Vector4(129.0f, 198.0f, 193.0f, 255.0f) / 255.0f));
-stopColor.Add(new PropertyValue(new Vector4(196.0f, 198.0f, 71.0f, 122.0f) / 255.0f));
-stopColor.Add(new PropertyValue(new Vector4(214.0f, 37.0f, 139.0f, 191.0f) / 255.0f));
-stopColor.Add(new PropertyValue(new Vector4(129.0f, 198.0f, 193.0f, 150.0f) / 255.0f));
-stopColor.Add(new PropertyValue(Color.Yellow));
+<a name="table_grad_possible_values_spread"></a>
+**Table: Possible values of the `SpreadMethod` property**
 
-gradientVisualMap1.StopColor = stopColor;
-gradientVisualMap1.Center = new Vector2(0.5f, 0.5f);
-gradientVisualMap1.Radius = 1.414f;
-gradientVisualMap1.Size = new Vector2(100.0f, 100.0f);
-gradientVisualMap1.Position = new Vector2(120.0f, 380.0f);
-gradientVisualMap1.PositionPolicy = VisualTransformPolicyType.Absolute;
-gradientVisualMap1.SizePolicy = VisualTransformPolicyType.Absolute;
-gradientVisualMap1.Origin = Visual.AlignType.TopBegin;
-gradientVisualMap1.AnchorPoint = Visual.AlignType.TopBegin;
+| Value               | Description                                                                                                  |
+|---------------------|--------------------------------------------------------------------------------------------------------------|
+| `Pad`               | Uses the terminal colors of the gradient to fill the remainder of the area.                                  |
+| `Reflect`           | Reflects the gradient pattern start-to-end, end-to-start, start-to-end, and so on, until the area is filled. |
+| `Repeat`            | Repeats the gradient pattern start-to-end, start-to-end, start-to-end, and so on, until the area is filled.  |
 
-_visualView.AddVisual("gradientVisual1", gradientVisualMap1);
-```
-<a name="imagevisual"></a>
-## Image Visual
 
-The image visual renders an image into the control's quad. There are different rendering visuals depending on the image extension:
 
--   [Normal (Quad) image](#normal)
--   [N-patch image](#npatch)
--   [SVG image](#svg)
--   [Animated image](#animated)
 
-The visual type for all of them is `Image`.
 
-<a name="normal"></a>
-### Normal (Quad) Image
+**Usage:**
+The following example illustrates how to set radial and linear `GradientVisual`s.
 
-The normal image visual renders a raster image (such as JPG or PNG) into the control's quad.
+Be aware of the difference in the radial gradients drawn for different `Units` values. 
+In case of `ObjectBoundingBox` value the given radius value is treated independently in the vertical and horizontal direction - for non squared visuals the resulting gradient will look like an ellipse (see case 1 below).
+For the value of `UserSpace`, the given radius is applied in all dimensions - it will always look like concentric circles (case 2 below).
```

Comment:
I changed a bit:
"... in the vertical and horizontal direction. For non-squared visuals the resulting gradient will look like an ellipse as shown in case 1:"